File: lab5v1.cpp
#include <iostream>
 
using namespace std;
 
int a, b, p, n, i;

int main(){

cout << "enter a: " ;
cin >> a;
cout << "enter b: " ;
cin >> b;

n=b-a;
 
if (n >= 0){
     cout << "p: " ;
    for (i=0; i<=n; i++)
     {
    cout << a+i << ", ";
    }
}
else{
cout << "Error! a is bigger than b!";
 }   
return 0;
}

File: lab5v1.py
a = float(input("a = "))
b = float(input("b = "))
n=b-a
i=0
if (n >= 0):
    print("p: ")
    while i <= n:
         print( float(a+i), str(", "))
         i += 1
else:
    print("Error! a is bigger than b!")